include/smack/SmackRep.h

```diff
@@ -83,7 +89,16 @@ class SmackRep {
   Program& getProgram() { return program; }
 
 private:
-  void addInit(unsigned region, const Expr* addr, const llvm::Constant* val);
+  // 
+  // Pointer size: 32
+  //
+  static const unsigned ptrsize;
+  //
+  // A flag for bit-vector
+  //
+  static bool BIT_VECTOR;
```

Comment:
Storing these two fields here is redundant, and inconsistent with the rest of our design. SMACK options are stored in and queried from SmackOptions, not SmackRep. Is there any reason to stray from that design?